source/administration/upgrade.rst

```diff
@@ -1,120 +1,75 @@
-
-Which Upgrade Instructions to Use
----------------------------------
-
-If you are upgrading from version 3.8.0 or greater
-  This guide applies to you. Skip to :ref:`Before you begin <before-you-begin>`.
-If you are upgrading from version 3.0.3 or greater and 3.7.5 or lower
-  Upgrade `Team Edition <../administration/legacy-upgrade.html#upgrade-team-edition-from-3-0-x-and-later>`_ or upgrade `Enterprise Edition <../administration/legacy-upgrade.html#upgrade-enterprise-edition-from-3-0-x-and-later>`_ to the latest version.
-If you are upgrading from version 2.2.0 or earlier
-  You must first upgrade to `Team Edition 3.0.3 <../administration/legacy-upgrade.html#upgrade-team-edition-to-3-0-x>`_ or if you have Enterprise Edition, upgrade to `Enterprise Edition 3.0.3 <../administration/legacy-upgrade.html#upgrade-enterprise-edition-to-3-0-x>`_.
-
-.. _before-you-begin:
-
-**Before you begin**
-
-Read these instructions carefully from start to finish. Make sure that you understand each step before starting the upgrade. If you have questions or concerns, you can ask on the Mattermost forum at https://forum.mattermost.org/ or in the Mattermost *Peer-to-peer Help* channel at https://pre-release.mattermost.com/core/channels/peer-to-peer-help.
-
-You should gather the following information before starting the upgrade:
-
-Existing install directory - *{install-path}*
-  If you don't know where Mattermost Server is installed, use the ``whereis platform`` command. The output should be similar to */opt/mattermost/bin/platform*. The install directory is everything before the last occurrence of the string */mattermost*. In this example, the *{install-path}* is ``/opt``.
-Location of your local storage directory
-  The local storage directory contains all the files that users have attached to their messages. If you don't know its location, open the System Console and go to **Files > Storage** and read the value in **Local Storage Directory**. Relative paths are relative to the ``mattermost`` directory. For example, if the local storage directory is ``./data/`` then the absolute path is ``{install-path}/mattermost/data``.
-Owner and group of the install directory - *{owner}* and *{group}*
-  Use the ``ls -l {install-path}/mattermost/bin/platform`` command to get the owner and group.
+In most cases you can upgrade Mattermost Server in a few minutes, but the upgrade can take longer depending on several factors, including the size and complexity of your installation, and the version that you're upgrading from.
+
+.. _upgrade-table:
+
+In the following table, look up your current version, review the applicable Important Notices, then click the link to the upgrade instructions.
+
+.. csv-table::
+  :header: "If your current version is:", "Read these Important Notices:", "Use these instructions:"
+  :widths: auto
+
+  "3.9.0", "Not Applicable", "`Upgrading to the latest version <../administration/upgrading-to-latest.html>`_"
```

Comment:
Has this table concept been reviewed from an "administrator" perspective? Are we basing this concept off of any examples in other software docs?